permissionless-batches/docker-compose.yml

```
@@ -0,0 +1,98 @@
+      - RUST_LOG=info
+    volumes:
+      - ./conf/proving-service/config.json:/prover/conf/config.json:ro
+      - ./conf/proving-service/prover_run.sh:/bin/prover_run.sh
+      - ./conf/proving-service/db:/db
+      - ./conf/proving-service/keys:/keys
+    depends_on:
+      coordinator-api:
+        condition: service_healthy
+
+volumes:
+    db_data:
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 2 but found 4

(indentation)